crates/typst-cli/src/args.rs

```diff
@@ -107,7 +108,7 @@ pub enum SerializationFormat {
 #[derive(Debug, Clone, Args)]
 pub struct SharedArgs {
     /// Path to input Typst file
-    pub input: PathBuf,
```

Comment:
the collision between the names here is a bit unfortunate ...

Reply:
Maybe "source" as the new name?